wwr_ui/build_ui.py

```diff
@@ -11,6 +11,6 @@
   command = [
     "pyside6-uic",
     input_path,
-    "-o", ui_dir + "uic/ui_%s.py" % base_name
+    "-o", ui_dir + "/uic/ui_%s.py" % base_name
```

Comment:
It might be worth merging this separately, even if not merging the rest of this PR? This looks like a bugfix

Reply:
fixed 94a622b0f9ffaa1116edfcd5d9e83820480deb24